src/app/data-model-provider/ActionContext.h

```diff
@@ -20,7 +20,7 @@
 
 namespace chip {
 namespace app {
-namespace InteractionModel {
+namespace DataModel {
 
 // Context for a currently executing action
 class ActionContext
```

Comment:
Actions are an interaction model concept, not a data model one...

Reply:
I can move them around. My typical expectation was that a folder generally operates under a single namespace, hence the bulk rename.

Should I just go ahead and move actioncontext and operationtypes into IM instead of DM? We would still then have the namespace conflict whenever we include these files (and they are included in interactionmodelengine, so often visible) ... however beyond that I don't have strong opinions, I just want some usable naming that is stable.

Reply:
Ah, hmm.  How about we merge this, then think about naming for these bits, then.  The name conflicts sound annoying.